src/core/utils/misc.cpp

```diff
@@ -226,13 +226,30 @@ QPoint Utils::Misc::screenCenter(QWidget *win)
 #endif
 
 /**
- * Detects the version of python by calling
- * "python --version" and parsing the output.
+ * Detects the python version.
  */
 int Utils::Misc::pythonVersion()
 {
     static int version = -1;
     if (version < 0) {
+        QString version_complete = pythonVersionComplete().trimmed();
```

Comment:
@sledgehammer999 It's really hard to remember when you already have `python_proc` 20 times in the next lines... I will do the changes but I don't wanna to rewrite the code of others each time I do a little fix.